modules/environment-roles/templates/shared_reference_generator_policy.json.tpl

```diff
@@ -9,7 +9,8 @@
       ],
       "Resource": [
         "arn:aws:iam::${account_id}:policy/DaReferenceCounterTableAccessPolicy${title(environment)}",
-        "arn:aws:iam::${account_id}:role/da*"
+        "arn:aws:iam::${account_id}:role/da*",
+        "arn:aws:iam::${account_id}:role/TDRTerraformRole${title(environment)}"
```

Comment:
This should be the policy not the role that will change the policy, ie: `arn:aws:iam::${account_id}:policy/DAReferenceGeneratorLambdaPolicy${title(environment)}`

Reply:
Sorry, changed that now.
I noticed the very top policy `DaReferenceCounterTableAccessPolicy${title(environment)}` should Da be all uppercase?

Reply:
Changed that now. For the `DaReferenceCounterTableAccessPolicy${title(environment)}`. Should Da be all uppercase? i.e.  `DAReferenceCounterTableAccessPolicy${title(environment)}`

Reply:
Yes, I'll get on to the dev who put the lowercase name in to fix it ;)